docs/developers/envvars.rst

```diff
@@ -0,0 +1,9 @@
+Environment Variables
+=====================
+
+This section documents all the environment variables supported by the client.
```

Comment:
The client _itself_ doesn't see or do anything with env vars. What you mean is that these env vars affect the behaviour of the client's build tasks.